roles/rsyslog/defaults/main.yml

```diff
@@ -154,20 +154,15 @@ rsyslog_send_permitted_peers: '{{ rsyslog_permitted_peers }}'
 # Firewall, UDP, TCP ports
 # ------------------------
 
-# rsyslog_udp_port
+# default rsyslog port
 #
-# The incoming UDP port used for remote logging.
-rsyslog_udp_port: '514'
+# The incoming TCP/UDP port used for remote logging.
+__rsyslog_default_port: '514'
```

Comment:
private variables not settable by users should go into `vars/main.yml` - https://github.com/oasis-roles/meta_standards#vars-vs-defaults

Reply:
This is a fallback port number used like this. Yes, I should move it to roles/rsyslog/vars/main.yml...
`input(name="{{ remote_item.name }}" type="imudp" port="{{ remote_item.udp_port | d(__rsyslog_default_port) }}")`